servers/src/main/java/tachyon/master/MasterContext.java

```diff
@@ -55,10 +55,19 @@ public static MasterSource getMasterSource() {
   }
 
   /**
-   * Reset the TachyonConf instance in master context, for test only.
+   * Resets the master context, for test only.
```

Comment:
Can we avoid adding a public methods that are used in tests only? I think we should be able to use PowerMock for this. (See BlockDataManagerTest for an example of extensive use of mocking).